mindsdb/integrations/handlers/autosklearn_handler/config.py

```diff
@@ -0,0 +1,34 @@
+import sys
+from dataclasses import dataclass
+from autosklearn.metrics import *
+from autosklearn.metrics import Scorer
+
+
+@dataclass(frozen=True)
+class ClassificationConfig:
```

Comment:
Seeing as both configs are, right now, identical, I wonder if it's best to define a `BaseConfig` where all shared params are defined so that they can be potentially overridden in the two existing classes, plus adding any task-specific params in them?

It's a nitpick, so feel free to ignore this one.

Reply:
No, this is an awesome suggestion. It's done!